angular/projects/catalyst/src/lib/catalyst.module.ts

```diff
@@ -17,7 +21,7 @@ const CatComponents = [
 export class CatalystModule {
 
   static forRoot(): ModuleWithProviders<CatalystModule> {
-    defineCustomElements();
+    defineCustomElements(window);
```

Comment:
I'm just wondering if can `defineCustomElements` be called without `window`?

Reply:
Same question. The examples I saw were calling the function without window?

Reply:
I guess it can work without it, I saw some examples that used window so I added it but I'll remove it.